docs/policy/service-migrations-spring-2018.md

```diff
@@ -159,6 +161,25 @@ service at this time.  The integration of OSG with WLCG accounting should see no
 If you experience any problems with GRACC accounting, please contact us at
 [help@opensciencegrid.org](mailto:help@opensciencegrid.org).
 
+### Ticket ###
+
+The Footprints ticketing system at <https://ticket.opensciencegrid.org> is used to track support and security issues as
+well as certificate and membership requests.
+This service will be retired in favor of two different ticketing systems, depending on the VOs you support at your site:
+
+| If your site primarily supports... | Submit new tickets to...                         |
+|------------------------------------|--------------------------------------------------|
+| Non-LHC VOs                        | [Freshdesk](https://support.opensciencegrid.org) |
```

Comment:
For the left column, I might suggest “LHC VOs” and “Anyone else”; that way, if you have no clue what the LHC is (or even what a VO is), you have a good chance of making the right decision. And then I would swap the rows so that they appear in the order I listed.